PowerShellTools/Classification/PowerShellParseUtilities.cs

```diff
@@ -187,6 +188,22 @@ internal static IList<ScriptParameterViewModel> GenerateCommonParameters()
                     new ScriptParameter("WarningVariable", DataTypeConstants.StringType, null, new HashSet<object>())
                     )
             };
+
+            if(DependencyUtilities.GetInstalledPowerShellVersion().Major >= 5)
+            {
+                // InformationAction
+                commonParameters.Insert(3, new ScriptParameterViewModel(

```

Comment:
We shouldn't be inserting based upon the index itself.  If this ends up with an unsorted list, then we should instead be sorting the list.